build/__tools__/run-tests.sh

```diff
@@ -147,6 +147,16 @@ try pdo-test-request --ledger ${PDO_LEDGER_URL} \
     --eservice http://localhost:7101/ \
     --logfile __screen__ --loglevel warn
 
+say start request test with tampered block order
+pdo-test-request --ledger ${PDO_LEDGER_URL} \
+    --pservice http://localhost:7001/ http://localhost:7002 http://localhost:7003 \
+    --eservice http://localhost:7101/ \
+    --tamper-block-order \
+    --logfile __screen__ --loglevel warn
+if [ $? == 0 ]; then
+    die request test with tampered block order succeeded though it should have failed
+fi
+
 say start integer-key contract test
```

Comment:
why put this here instead of up above where we don't have an eservice?

Reply:
I moved it up -- though it might be good to have this test with and without eservice, because it would follow two different execution paths.

Reply:
we're only testing the eservice, not test-requests. also... you're expecting the failure, but could it just be failing because it simply crashed? how do you know it was the HMAC?